docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -160,52 +160,106 @@ This example will produce the following output:
 
 Jobs are created when a task is submitted.
 
-- You can cancel the Job with `taipy.cancel_job(job)^`. When cancelling a Job, you will effectively set the `Status^` of subsequent jobs of the cancelled Job to `ABANDONED`.
+- You can cancel a Job with `taipy.cancel_job(job)^`. When canceling a Job, you will effectively set the `Status^` of subsequent jobs of the canceled Job to `ABANDONED`. However, a job cannot be canceled if it is already running. When canceling a running job, its subsequent jobs will be abandoned and the running job will continue running until completed.
 
-!!! Warning
-    Cancelling a `RUNNING` Job will set the Job to `CANCELLED`. However, it will not terminate the subprocess that has already been dispatched to execute the Job.
+!!! example "Canceling a job"
 
-!!! Example
+    ```python
+    import taipy as tp
 
-    ```python linenums="1"
+    def double(nb):
+        sleep(5)
+        return nb * 2
+
+    print(f'(1) Number of jobs: {len(tp.get_jobs())}.')
+
+    # Create a scenario then submit it.
+    input_data_node_config = tp.configure_data_node("input", default_value=21)
+    output_data_node_config = tp.configure_data_node("output")
+    double_task_config = tp.configure_task("double_task", double, input_data_node_config, output_data_node_config)
+    print_task_config = tp.configure_task("print_task", print, output_data_node_config)
+    scenario_config = tp.configure_scenario_from_tasks("my_scenario", [double_task_config, print_task_config])
+    scenario = tp.create_scenario(scenario_config)
+    tp.submit(scenario)
+
+    # Retrieve all jobs.
```

Comment:
This comment should be something like:
    Count the Jobs.
    
 (as you have used capitals in the whole text)

Reply:
I'll go for "Count the jobs" :D